README.md

```diff
@@ -1,13 +1,11 @@
 # Project Movies
 
-Replace this readme with your own information about your project.
+Project of Jessica Hermansson and Aschwin Siaila
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The project started good Jessica did a lot of the framework. Then we started to work together in VS-code we seperated the tasks and were able to merge the codes. The project was working with react-router and we were able to manage the requirements.
 
-## The problem
+We had some challenges with the response of the API and also to catch the errors, but it took us not to long to figure this out. At the end we styled and cleaned up the code to finish the project.
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Watcht the result in:
 
```

Comment:
Hej guys, we think that the Read.me-file could be more general and focused to the project. Keep in mind, that it can be read by future recruiters, so maybe add more technical details about the project and what it was about. Instead of writing who did what, you could just state that this was a pair-programming-project.